src/routers/user-router.ts

```diff
@@ -0,0 +1,130 @@
+import { Request, Response, Router } from 'express';
+import { v4 as uuidv4 } from 'uuid';
+
+import asyncHandler from '../handle-midleware/utils';
+import { validateSchema } from '../validation/validation-middleware';
+
+import * as users from '../../data/data.json';
+import { userSchema } from '../validation/schema';
+
+export const router = Router();
+
+export interface User {
+    id: string;
+    login: string;
+    password: string;
+    age: number;
+    isDeleted: boolean;
+}
+
+const getAutoSuggestUsers = (loginSubstring: string, limit: string) =>
+    users.users
+        .filter((user: User) => !user.isDeleted)
```

Comment:
We do not have any AC for that condition, anyway for me it is not a problem.

Reply:
This is logical, we should not receive users if they were deleted.